tests/test_ramview_counts.C

```diff
@@ -0,0 +1,63 @@
+#include <TSystem.h>
+#include <cassert>
+#include <cstdio>
+#include <cstring>
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+
+void test_ramview_counts(const char* samfile = "samexample.sam",
+                         const char* region  = "chr1:1-10000000") {
+    const char* ttreeFile   = "ttree_view.root";
+    const char* rntupleFile = "rntuple_view.root";
+
+    gSystem->Unlink(ttreeFile);
+    gSystem->Unlink(rntupleFile);
+    gSystem->Unlink("ttree_out.txt");
+    gSystem->Unlink("rntuple_out.txt");
+
+    int r1 = gSystem->Exec(Form("root -l -b -q 'samtoram.C+(\"%s\", \"%s\")'", samfile, ttreeFile));
```

Comment:
Outside of this PR we should probably add googletest and google benchmark to have a more extensive way to study performance. You can take a look how it is done in the clad project.